doc/prosoul-grimoirelab.md

```diff
@@ -1,39 +1,144 @@
+        ]
+}
 ```
 
-Once the script execution has finished, you have in Elasticsearch the projects data needed for
-doing the assessment and visualization of them using ProSoul.
+setup.cfg
+```
+[gitqm]
+raw_index = grimoirelab_gitqm_raw
+enriched_index = grimoirelab_gitqm_enriched
+category = commit
+
+[githubqm:issue]
+raw_index = grimoirelab_githubqm_issues_raw
+enriched_index = grimoirelab_githubqm_issues_enriched
+api-token = xxxx
+sleep-for-rate = true
+sleep-time = 300
+category = issue
+no-archive = true
+```
+You can find all the configurations here, link.
+
+Now, execute the raw and enrich tasks of micro-mordred.
+```
+micro.py --raw --enrich --cfg ./setup.cfg --backends gitqm githubqm:issue
+```
+
+Setup alias for all the enriched indices.
+```
+POST /_aliases
+{
+  "actions": [
+    {
+      "add": {
+        "indices": [
+          "grimoirelab_gitqm_enriched",
+          "grimoirelab_githubqm_issues_enriched"
+        ],
+        "alias": "all_qm_data"
+      }
+    }
+  ]
+}
+```
+
+Now, you have all the required enriched data in `qm-enriched-data` index which is needed for the assessement and visualization using Prosoul.
 
 ### Import the QualityModel
 
-* Import the [Developer Quality Model](https://github.com/Bitergia/prosoul/blob/master/django_prosoul/prosoul/data/developer_model.json) using prosoul web editor
+Import the [Developer Quality Model](https://github.com/Bitergia/prosoul/blob/master/django_prosoul/prosoul/data/developer_model_grimoirelab.json) using prosoul web editor.
+
+### Load the Metrics Data
 
-### Create the Visualization and Assessment using ProSoul web interface
+Login with the admin credentials and go to the admin panel, `Prosoul Site Administration` and click on `Metric datas`. You have to entry the metric items data by clicking on the `ADD METRIC DATA` button. You can add the metric data by filling out the web form using the `metric_name`.
 
-Use the Visualize and Assess link in Prosoul web interface to generate them. 
+- Description: Number of Commits
+- Implementation: Number of Commits
+
+Similarly, you have to add all the metrics data which you use while making a Quality Model.
+
+### Create the Assessment
+
+The next step is to create the assessment using prosoul assessment web form.
+
+- Quality Model: Developer Quality Model
+- Elasticsearch URL: https://localhost:9200/
+- Index with metrics data: all_qm_data
+
+Click on `Create` and the assessment will be completed in a while.
+
+After the assessment is completed, you can view the results using the prosoul web interface.
+
+### Import the Dashboard
+
+Setup alias for all the qm results indices.
+```
+POST /_aliases
+{
+  "actions": [
+    {
+      "add": {
+        "indices": [
+          "all_qm_data_null_scores",
+          "all_qm_data_null_scores_by_quarters",
+          "all_qm_data_scores",
+          "all_qm_data_scores_by_quarters"
+        ],
+        "alias": "all_qm_data_results"
+      }
+    }
+  ]
+}
+```
```

Comment:
Would it be good if we can automate the alias adding?

I understand that we can add an alias to the prosoul indices. [prosoul_assess.py#L754](https://github.com/Bitergia/prosoul/blob/master/django-prosoul/prosoul/prosoul_assess.py#L754)

So, we can add a common alias (`all_results_alias`) to all the prosoul indices.

```
all_results_alias = es_index + "_results"
# all_results_alias = all_qm_data_results

es_conn.indices.update_aliases({
    "actions": [
        {"add": {"index": null_scores_index, "alias": all_scores_alias}},
+       {"add": {"index": null_scores_index, "alias": all_results_alias}},
        {"add": {"index": scores_index, "alias": all_scores_alias}},
+       {"add": {"index": scores_index, "alias": all_results_alias}},
        {"add": {"index": null_scores_quarters_index, "alias": all_scores_quarters_alias}},
+       {"add": {"index": null_scores_quarters_index, "alias": all_results_alias}},
        {"add": {"index": scores_quarters_index, "alias": all_scores_quarters_alias}},
+       {"add": {"index": scores_quarters_index, "alias": all_results_alias}},
    ]
})
```

WDYT?